packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_counter/views/counter_page.dart

```diff
@@ -39,13 +31,23 @@ class CounterPage extends StatelessWidget implements AutoRouteWrapper {
                 builder: (context, countState, bloc) =>
                     _buildCount(context, countState),
               ),
-              RxBlocListener<CounterBlocType, ErrorModel>(
-                state: (bloc) => bloc.states.errors,
-                listener: _onError,
+              AppErrorModelWidget<CounterBlocType>(
+                errorState: (bloc) => bloc.states.errors,
+              ),
+              AppErrorModelWidget<UserAccountBlocType>(
+                errorState: (bloc) => bloc.states.errors,
+                isListeningForNavigationErrors: false,
               ),
-              RxBlocListener<UserAccountBlocType, ErrorModel>(
-                state: (bloc) => bloc.states.errors,
-                listener: _onError,
+              RxBlocListener<CounterBlocType, String>(
+                state: (bloc) => bloc.states.message,
+                condition: (old, nee) => (old != nee && nee.isNotEmpty),
+                listener: (context, message) =>
+                    ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
Please extract this closure as a private method or some reusable global function

Reply:
Done